iOSClient/Viewer/NCViewerMedia/NCPlayer/NCPlayer.swift

```diff
@@ -105,10 +105,7 @@ class NCPlayer: NSObject {
 #endif
 
         // Check already started
-        if isStartPlayer {
-            activateObserver()
-            return
-        }
+        if isStartPlayer { return }
```

Comment:
is this related to the multi-account issue?

Reply:
hahaah (I knew) was an refused, you can approved :)